src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -108,10 +125,27 @@ export class CheckBox extends React.Component<Props> {
     ];
   };
 
+  private renderSelectIcon = (style: StyleType): React.ReactNode => {
```

Comment:
prefer wrapping jsx into parenthesis in `render` methods
like you do in core `render`

Reply:
Done